android_automation/page_action/category_page.py

```diff
@@ -60,3 +62,145 @@ def click_back_btn(wd):
     # 뒤로가기로 카테고리 화면 진입
     top_menu = aal(wd, 'com.the29cm.app29cm:id/topMenu')
     aalc(top_menu, '//android.view.View/android.view.View')
+
+
+def check_large_category_list(wd, api_large_categoty_list):
+    # api에서 호출한 리스트 길이와 비교하여 노출되는 대 카테고리 리스트 저장
+    large_list = []
+    category_layer = aal(wd, 'com.the29cm.app29cm:id/shopComposeView')
+
+    large_list.append(aal(wd, 'category_first_title').text)
+    large_list.append(
+        aal(category_layer, '//android.view.View/android.view.View[2]/android.widget.TextView[2]').text)
+    large_list.append(
+        aal(category_layer, '//android.view.View/android.view.View[2]/android.widget.TextView[3]').text)
+
+    if set(large_list).intersection(api_large_categoty_list):
+        print('대 카테고리 리스트 확인')
+    else:
+        print(f'대 카테고리 리스트 확인 실패 - {api_large_categoty_list} / {large_list} ')
+        raise Exception('대 카테고리 리스트 확인 실패')
+
+
+def scroll_up_large_category(wd):
+    large_field = aal(wd, 'large_category_list')
+    element_scroll_control(wd, large_field, 'U', 30)
+
+
+def click_category(wd, category_name):
+    category_layer = wd.find_element(AppiumBy.ID, 'com.the29cm.app29cm:id/shopComposeView')
+    aalc(category_layer, category_name)
+
+
+def check_category_page_shose_title(wd, category_name):
+    sleep(3)
+    page_title = aal(wd, 'com.the29cm.app29cm:id/txtCategoryName')
+    if page_title == None:
+        print(f'{category_name} 카테고리 PLP 진입 확인 실패qq')
+        raise Exception(f'{category_name} 카테고리 PLP 진입 확인 실패qq')
+    elif '신발' in page_title.text:
+        print('카테고리 전체 페이지 진입 확인')
+        print(f'{category_name} 카테고리 PLP 진입 확인')
+    else:
+        print(f'{category_name} 카테고리 PLP 진입 확인 실패22')
+        raise Exception(f'{category_name} 카테고리 PLP 진입 확인 실패')
```

Comment:
위 코드 아래처럼 수정할 수 있지 않을까요??
```suggestion
        if '신발' in page_title.text:
           print('카테고리 전체 페이지 진입 확인')
           print(f'{category_name} 카테고리 PLP 진입 확인')
       else:
           print(f'{category_name} 카테고리 PLP 진입 확인 실패')
           raise Exception(f'{category_name} 카테고리 PLP 진입 확인 실패')
```

Reply:
넵 제안해 주신 코드로 수정했습니다. 확인 감사합니다!